tests/trainer/test_dataloaders.py

```diff
@@ -635,6 +633,62 @@ def test_warning_with_few_workers(mock, tmpdir, ckpt_path):
         trainer.test(**test_options)
 
 
+@pytest.mark.skipif(platform.system() == 'Windows', reason='Does not apply to Windows platform.')
+@pytest.mark.parametrize('ckpt_path', [None, 'best', 'specific'])
+@patch('pytorch_lightning.trainer.data_loading.multiprocessing.cpu_count', return_value=4)
+def test_warning_with_few_workers_multi_loader(mock, tmpdir, ckpt_path):
+    """ Test that error is raised if dataloader with only a few workers is used """
+
+    model = EvalModelTemplate()
```

Comment:
use BoringModel?

Reply:
@rohitgr7 I revisited this and for consistency I opted for sticking with `EvalModelTemplate` as this is also what's used for other loader tests (with various defined functions that would only blow up the Boring Model)